### DISCUSSION TITLE
Charging threshold don't work

### DISCUSSION THREAD

**Discussion by Glexu** [flobz/psa_car_controller#894] [Q&A]
Does charging threshold work for you? Because I can't get it to work. 

I have PSACC 3.5.0. It shows all charging data, current state of the battery, saves trip paths. Unfortunately, the charging limit I set in the charge_config.json file does not work at all. 

Am I doing something wrong? Is it the fault of the new API? Who can help me?

**Comment by ricardolobo81** [comment]
Same thing here.
Before version 3.5.0, I was able to use it and it worked fine.
Since version 3.5.0...3.5.1, it no longer works.
Can anyone help?
Thanks.
